plugins/modules/system/parted.py

```diff
@@ -500,25 +500,22 @@ def check_parted_label(device):
     return False
 
 
-def parted_version():
+def parted_fetch_version(out):
```

Comment:
I think naming this `parse_parted_version()` would be a better description of what it does.

Reply:
thanks